R/deprecate.R

```diff
@@ -102,18 +102,27 @@ deprecate_soft <- function(when,
   signal_stage("deprecated", what)
 
   verbosity <- lifecycle_verbosity()
-  if (verbosity == "quiet") {
-    NULL
-  } else if (verbosity %in% c("warning", "default")) {
-    if (is_direct(user_env)) {
-      always <- verbosity == "warning"
-      deprecate_warn0(msg, id, trace_back(bottom = caller_env()), always = always)
-    }
-  } else if (verbosity == "error") {
-    deprecate_stop0(msg)
-  }
-
-  invisible(NULL)
+  direct <- is_direct(user_env)
+
+  invisible(switch(
+    verbosity,
+    quiet = NULL,
+    error = deprecate_stop0(msg),
```

Comment:
I think this should go at the bottom so `verbosity` is order from quietest to loudest.